2789 ZAP Search Project Page - Update milestone display name

dhochbaum-dcp opened the pull request:
<!---
   The below template is a general guide, but not a strict prescription!
-->

### Summary
This adds two new milestones, and updates how two other milestones names to normalize with names the DCP planners see. 

#### Tasks/Bug Numbers
 - Implements [AB#2789](https://dev.azure.com/NYCPlanning/ITD/_workitems/edit/2789)
<!---

### Technical Explanation
<!---
  a. In technical terms, what was wrong, what is the fix, and why does it make things better? 
  b. If you can point to a specific line or file exhibiting the original problem, that would be great!
  c. Provide entrypoint of new solution, if different than (b).
  d. Provide overview of any new architecture.
       - How are components stringed together?
       - What is the new hierarchy? 
       - Any new components?
  e. Provide links and explanations for any new technical concepts, APIs and terms.
      Especially if you had to do some research yourself.
   List any ad-hoc, miscellaneous updates included
-->

### Any other info you think would help a reviewer understand this PR?

allthesignals commented:
I think the auto eslint is obscuring the focus of the PR... even going commit-by-commit, I see a lot of eslint auto fixes but have no idea which changes are you made by you and which are made by eslint... 

I thought our first mistake with auto-listing was to forget to do an initial listing cleanup. That said, I thought it was only applying auto-linting to code found in the diff...

dhochbaum-dcp commented:
> I think the auto eslint is obscuring the focus of the PR... even going commit-by-commit, I see a lot of eslint auto fixes but have no idea which changes are you made by you and which are made by eslint...
> 
> I thought our first mistake with auto-listing was to forget to do an initial listing cleanup. That said, I thought it was only applying auto-linting to code found in the diff...

Definitely wish it hadn't done the linting as well.  The things I did were:

- Did a find and replace on the two existing variable names ("CPC Review of Modification Scope" on line 65 of constants.ts becomes "City Planning Commission Review of City Council Modification" on line 160.  
- Added the other three to the Allowed Milestones on lines 161-163.  
- Added two new milestones on 286-291
- In transform-milestones.ts, once again did a find and replace for existing variables.
- Added two new variables on lines 163-166.
